src/X.PagedList/BasePagedList.cs

```diff
@@ -20,16 +20,23 @@ namespace X.PagedList;
 [PublicAPI]
 public abstract class BasePagedList<T> : IPagedList<T>
 {
-    protected readonly List<T> Subset = new();
-
     public const int DefaultPageSize = 100;
 
+    private readonly List<T> _subset = new();
```

Comment:
I understand what you are doing here, but why do you do it?

It's just a list, and you do not do any additional actions in FillSubset; derived classes should be capable of dealing with it.